module/account/migrations/Version20201202075632.php

```diff
@@ -0,0 +1,187 @@
+<?php
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\Migration;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Schema\Schema;
+use Ramsey\Uuid\Uuid;
+
+/**
+ * Auto-generated Ergonode Migration Class:
+ */
+final class Version20201202075632 extends AbstractErgonodeMigration
+{
+    public function up(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE privileges_group RENAME TO privileges_family');
+        $this->addSql('ALTER TABLE "privileges" RENAME TO privileges_group');
+        $this->addSql('ALTER INDEX privileges_name_key RENAME TO privileges_group_code_key');
+
+        $this->addSql(
+            '
+            CREATE TABLE privileges (
```

Comment:
I am thinking about whether or not to name a new table `privilege_endpoint`?

Because are a lot of technical changes in this path.